common/views/components/Divider/Divider.tsx

```diff
@@ -1,11 +1,50 @@
 import { FunctionComponent, ReactElement } from 'react';
+import styled from 'styled-components';
 
 type Props = {
-  extraClasses?: string;
+  color: string;
```

Comment:
should we make this type more specific, as in the colours in the theme

Reply:
we should. we need to do this for the colour function as well. I'll open a separate pr for both if that's ok?